packages/ui/src/forms/Form/fieldReducer.ts

```diff
@@ -263,6 +263,15 @@ export function fieldReducer(state: FormState, action: FieldAction): FormState {
         duplicateRowState.id.initialValue = new ObjectId().toHexString()
       }
 
+      for (const key of Object.keys(duplicateRowState).filter((key) => key.endsWith('.id'))) {
+        const idState = duplicateRowState[key]
+
+        if (idState && typeof idState.value === 'string' && ObjectId.isValid(idState.value)) {
+          duplicateRowState[key].value = new ObjectId().toHexString()
+          duplicateRowState[key].initialValue = new ObjectId().toHexString()
```

Comment:
There is likely no observable difference, but shouldn't these variables be assigned the same value?

Reply:
I had the same question here
https://github.com/payloadcms/payload/blob/ef8a5b1f3e94a4d12b96cee226ff0857cb33e887/packages/ui/src/forms/Form/fieldReducer.ts#L260-L264
So I just did how it was there

Reply:
Should be fine then. Seems odd to me, but it probably can't hurt anything!